lib/compute/agent-nodes.ts

```diff
@@ -28,7 +28,9 @@ export class AgentNodes {
 
   readonly UBUNTU2004_X64_GRADLE_CHECK: AgentNodeProps;
```

Comment:
Do we want to replace the previous agent?

Reply:
Nope, we will keep it for now as gradle check has a bigger impact.
We will bake 2404 for a few days before completely replace.

Thanks.